ci-operator/populate-secrets-from-bitwarden.sh

```diff
@@ -16,6 +16,25 @@ set -o xtrace
 #          content of the BitWarden vault after your current session was started,
 #          you will need to create a new session to be able to view those changes.
 
+readonly CURRENT_OC_CONTEXT=$(oc config current-context)
+readonly OC_PROJECT=$(echo ${CURRENT_OC_CONTEXT} | cut -d "/" -f1)
+readonly OC_CLUSTER=$(echo ${CURRENT_OC_CONTEXT} | cut -d "/" -f2)
+readonly OC_USER=$(echo ${CURRENT_OC_CONTEXT} | cut -d "/" -f3)
+
+if [[ "${OC_CLUSTER}" != "api-ci-openshift-org:443" ]]; then
+	echo "current cluster ${OC_CLUSTER} is not our ci-cluster ... please run 'oc login https://api.ci.openshift.org' first!"
+	exit 1
+fi
+
+if [[ "${OC_PROJECT}" != "ci" ]]; then
+	echo "Warning: current project ${OC_PROJECT} is not 'ci'!"
+fi
+
+if [[ "yes" != "$(oc auth can-i create secrets -n ${OC_PROJECT})" ]]; then
```

Comment:
There is a `--quiet` flag that gives an error code instead of text output.

Reply:
Done.